wandb/sdk/launch/environment/gcp_environment.py

```diff
@@ -0,0 +1,222 @@
+"""Implementation of the GCP environment for wandb launch."""
+import logging
```

Comment:
Yes, logging, please add logging to the AWS environment

Reply:
Done :) added logging to the registries as well